specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-09-02-preview/examples/Usages_List.json

```diff
@@ -0,0 +1,33 @@
+{
+  "parameters": {
+    "api-version": "2023-09-02-preview",
+    "subscriptionId": "subid1",
+    "location": "location1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "currentValue": 8,
+            "limit": 1000,
+            "name": {
+              "localizedValue": "Managed Clusters",
```

Comment:
minor: should be consistent, below you do `Number of FleetHubs` so this should probably be `Number of Managed Clusters`?

Reply:
update on fleet one instead of this one. 

since count is already express in unit, so I am removing Number of/Count of.